packages/core/src/application.ts

```diff
@@ -121,15 +123,24 @@ export class Application extends Context {
     request: ServerRequest,
     response: ServerResponse,
   ) {
-    if (request.method === 'GET' && request.url === '/openapi.json') {
+    if (request.method === 'GET' &&
+      (request.url === '/openapi.json' ||
+       request.url === '/swagger.json' ||
+       request.url === '/openapi.yaml' ||
+       request.url === '/swagger.yaml')
+      ) {
```

Comment:
http://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if

I'd personally avoid multiple places checking `request.url` and rewrite this whole block using a simple lookup table.

```ts
// at the top
const SPEC_ENDPOINTS = {
  '/openapi.json': {version: '3.0.0', format: 'json'},
  // ...
  '/swagger.yaml': {version: '2.0', format: 'yaml'},
};

// here
if (request.method === 'GET') {
  const specOptions = SPEC_ENDPOINTS[request.url];
  if (specOptions) {
    return this._serveOpenApiSpec(request, response, specOptions);
  }
}
```